kitsune/settings.py

```diff
@@ -586,15 +599,13 @@
 
 
 def JINJA_CONFIG():
```

Comment:
Puente can extract the list of Jinja2 extensions to use from the old jingo style (i.e. this ^^^).

https://github.com/mozilla/puente/blob/master/puente/commands.py#L14

However, I'd recommend ditching it and adding the appropriate things to the PUENTE config.